content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -793,6 +799,15 @@ This system is the account number of a financial account.
       "value": "5646"
     }
 
+##### Bill Codes Types
+
+This system is the bill code type for a charge item.`Bill code type` can be either `CDM_SCHED`, `CPT`, `HCPCS`, `MODIFIER` or `REVENUE`.
```

Comment:
I thought we still supported ICD Bill Code Types in the reason field.

Reply:
Yes. Thanks for catching. Added [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/64ddf564c76cd6938adb0bdff00feda5cff09c86#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R804)

Reply:
Thanks!